packages/mesos/extra/dcos-mesos-master.service

```diff
@@ -15,4 +15,4 @@ EnvironmentFile=-/run/dcos/etc/mesos-master
 ExecStartPre=/bin/ping -c1 ready.spartan
 ExecStartPre=/bin/bash -c 'for i in /proc/sys/net/ipv4/conf/*/rp_filter; do echo 2 > $i; echo -n "$i: "; cat $i; done'
 ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-mesos-master
-ExecStart=${PKG_PATH}/bin/mesos-master
+ExecStart=$PKG_PATH/bin/tee_stderr $PKG_PATH/bin/mesos-master /var/lib/dcos/mesos/log/mesos-master.log
```

Comment:
`MESOS_EXTERNAL_LOG_FILE` modifies the link that shows up in the (somewhat obscure) `LOG` link in the Mesos web UI.  When `MESOS_LOG_DIR` is set, the link refers to the glog file.